v0/destinations/ga4/utils.js

```diff
@@ -357,8 +358,42 @@ const getGA4CustomParameters = (message, keys, exclusionFields, payload) => {
 const responseHandler = (destinationResponse, dest) => {
   const message = `[GA4 Response Handler] - Request Processed Successfully`;
   let { status } = destinationResponse;
+  const { response } = destinationResponse;
   if (status === 204) {
+    // GA4 always returns a 204 response, other than in case of
+    // validation endpoint.
     status = 200;
+  } else if (
+    // for GA4 debug validation endpoint, status is always 200
```

Comment:
we shouldn't put comments on the condition itself. Put it after the condition i.e on line 372

Reply:
done.